### ISSUE TITLE
Support Windows ARM64

### ISSUE THREAD

**Issue by dennisameling** [ONLYOFFICE/DesktopEditors#988]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my idea.

### Your idea.

The Desktop Editors [are already available for Apple Silicon/ARM](https://www.onlyoffice.com/download-desktop.aspx?from=desktop), but not for Windows yet. Will Windows ARM64 be supported and if no, could you please provide compilation instructions on how users can compile it themselves? Happy to support in the development. Currently, there's [only compilation instructions for Ubuntu](https://helpcenter.onlyoffice.com/installation/desktop-compile.aspx).

Thanks in advance!

**Comment by ShockwaveNN** [comment]
Hi, we already have issue 57191 in our internal issue tracker to add Windows ARM support

Also, we still have no proper instruction (this [issue](https://github.com/ONLYOFFICE/build_tools/issues)) and sorry, we cannot guarantee any time frame when it will be done

**Comment by jamroks** [comment]
Hi, 

 Is there any update for the Windows ARM Support ? 

Kind regards

**Comment by Rita-Bubnova** [comment]
@jamroks Sorry, no news.

**Comment by dennisameling** [comment]
@Rita-Bubnova If you could provide [compilation instructions for Windows](https://github.com/ONLYOFFICE/build_tools/issues/105), I'd be happy to have a look into this 😊

**Comment by Rita-Bubnova** [comment]
Sorry, I'm only a QA specialist and I can't help with this.

**Comment by alfredonodo** [comment]
Hi,
do you have any updates on this? Especially on GNU/Linux, as windows since version 11 can [emulate any amd64 application](https://learn.microsoft.com/en-us/windows/arm/arm64ec).
Thanks

**Comment by Dinkh** [comment]
I am currently running the x64 version on my Aarch64 windows device. Unfortunately I cannot update the version, as the installer seems to have a problem (another version is installed, cannot update, please uninstall ...).
I do not want to loose my settings, so I hoped for a native supported version. Seems nothing is happing for 2 years.

Any update available?

**Comment by gaussandhisgun** [comment]
So, still no news on this? As ARM-powered "Copilot+" PCs become more popular (and less usable), having ARM64 builds of Onlyoffice might provide a viable alternative for those who can't be bothered to deal with Microsoft Issues, both on Linux and Windows.

**Comment by teobugslayer** [comment]
Please create an Windows on Arm build! The x64 version works but is slow and lagging.

**Comment by HathiLuke** [comment]
I agree with all people here! An ARM native version will be widely appreciated and it will spread OnlyOffice market

**Comment by talynone** [comment]
Windows 11 ARM downloads are now available : https://www.onlyoffice.com/download-desktop

This issue should be closed.

**Comment by dennisameling** [comment]
Neat! Looks like OnlyOffice started supporting Windows on ARM in version 9.2.0, which was released on Dec 1st. https://github.com/ONLYOFFICE/DesktopEditors/releases/tag/v9.2.0

**Comment by Rita-Bubnova** [comment]
Sorry, we forgot to mention that issue was fixed.
I close this issue. Feel free to comment or reopen it if you got further questions.
